operator/src/main/java/org/bf2/operator/managers/InformerManager.java

```diff
@@ -104,6 +107,7 @@ protected void handleEvent(HasMetadata resource, ResourceAction action) {
 
         if (isOpenShift()) {
             routeInformer = resourceInformerFactory.create(Route.class, filterManagedByFleetshardOrStrimzi(openShiftSupport.adapt(kubernetesClient).routes()), eventSource);
+            infrastructureInformer = resourceInformerFactory.create(Infrastructure.class, openShiftSupport.adapt(kubernetesClient).config().infrastructures().withName("cluster"), eventSource);
```

Comment:
Unless there's a owner of the infrastructure, we don't want pass the eventSource here.

Reply:
Removed the informer with the latest commit.